db/migrate/20161001090839_create_flags.rb

```diff
@@ -0,0 +1,11 @@
+class CreateFlags < ActiveRecord::Migration
+  def change
+    create_table :flags do |t|
+      t.integer :post_id
+      t.integer :generated_topic_id
+      t.text :reason
+
+      t.timestamps null: false
```

Comment:
I think timestamps should be included because it might let us use some kind of time decay counter in the future to automatically disable posts that have been flagged many times recently.